e2e-tests/tfm/cypress/e2e/partials/index.js

```diff
@@ -3,11 +3,13 @@ const caseSummary = require('./caseSummary');
 const header = require('./header');
 const underwritingSubNav = require('./underwritingSubNav');
 const primaryNavigation = require('./primaryNavigation');
+const clickEvents = require('./clickEvents');
 
 module.exports = {
   caseSubNavigation,
   caseSummary,
   header,
   underwritingSubNav,
   primaryNavigation,
+  clickEvents,
```

Comment:
Can we change this to simply be this please:

```js
backLink: () => cy.get('[data-cy="back-link"]'),
```

So that TFM E2E will be aligned with the same structure as GEF/Portal (and 2x other incoming PRs)

Then we can remove `e2e-tests/tfm/cypress/e2e/partials/clickEvents.js`